rocAL/rocAL/source/master_graph.cpp

```diff
@@ -919,6 +1004,8 @@ void MasterGraph::output_routine()
                 _process_time.end();
             }
 
+            _sequence_start_framenum_vec.insert(_sequence_start_framenum_vec.begin(), full_batch_sequence_start_frame_number);
```

Comment:
get rid of full_batch_sequence_start_frame_number and full_batch_sequence_frame_timestamps. Use the sequence_start_framenum and sequence_frame_timestamp directly.

Reply:
Sure we will remove those variables.